evalml/tests/component_tests/test_xgboost_classifier.py

```diff
@@ -74,3 +74,13 @@ def test_xgboost_multiindex(data_type, X_y_binary, make_data_type):
     y_pred_proba = clf.predict_proba(X)
     assert not y_pred.isnull().values.any()
     assert not y_pred_proba.isnull().values.any().any()
+
+
+def test_xgboost_predict_all_boolean_columns():
+    X = pd.DataFrame({"a": [True, False, True], "b": [True, False, True]})
```

Comment:
This is the rare case where black preferred to un-lengthen my original definition:

```python
pd.DataFrame({"a": [True, False, True],
                           "b": [True, False, True]})
```

Reply:
I love it